REVIEWERS.md

```diff
@@ -6,154 +6,154 @@ Two +1 votes by any of these module reviewers on a new module pull request will
+-  Joseph Callen [@jcpowermac](https://www.github.com/jcpowermac) 
+-  Kevin Carter [@cloudnull](https://www.github.com/cloudnull) 
+-  Lester Wade [@lwade](https://www.github.com/lwade) 
+-  Lorin Hochstein [@lorin](https://www.github.com/lorin) 
+-  Manuel Sousa [@manuel-sousa](https://www.github.com/manuel-sousa) 
+-  Mark Theunissen [@marktheunissen](https://www.github.com/marktheunissen) 
+-  Matt Coddington [@mcodd](https://www.github.com/mcodd) 
+-  Matt Hite [@mhite](https://www.github.com/mhite) 
+-  Matt Makai [@makaimc](https://www.github.com/makaimc) 
+-  Matt Martz [@sivel](https://www.github.com/sivel) 
+-  Matt Wright [@mattupstate](https://www.github.com/mattupstate) 
+-  Matthew Vernon [@mcv21](https://www.github.com/mcv21) 
+-  Matthew Williams [@mgwilliams](https://www.github.com/mgwilliams) 
+-  Matthias Vogelgesang [@matze](https://www.github.com/matze) 
+-  Max Riveiro [@kavu](https://www.github.com/kavu) 
+-  Michael Gregson [@mgregson](https://www.github.com/mgregson) 
+-  Michael J. Schultz [@mjschultz](https://www.github.com/mjschultz) 
+-  Michael Schuett [@michaeljs1990](https://www.github.com/michaeljs1990) 
+-  Michael Warkentin [@mwarkentin](https://www.github.com/mwarkentin) 
+-  Mischa Peters [@mischapeters](https://www.github.com/mischapeters) 
+-  Monty Taylor [@emonty](https://www.github.com/emonty) 
+-  Nandor Sivok [@dominis](https://www.github.com/dominis) 
+-  Nate Coraor [@natefoo](https://www.github.com/natefoo) 
+-  Nate Kingsley [@nate-kingsley](https://www.github.com/nate-kingsley) 
+-  Nick Harring [@NickatEpic](https://www.github.com/NickatEpic) 
+-  Patrick Callahan [@dirtyharrycallahan](https://www.github.com/dirtyharrycallahan) 
+-  Patrick Ogenstad [@ogenstad](https://www.github.com/ogenstad) 
+-  Patrick Pelletier [@skinp](https://www.github.com/skinp) 
+-  Patrik Lundin [@eest](https://www.github.com/eest) 
+-  Paul Durivage [@angstwad](https://www.github.com/angstwad) 
+-  Pavel Antonov [@softzilla](https://www.github.com/softzilla) 
+-  Pepe Barbe [@elventear](https://www.github.com/elventear) 
+-  Peter Mounce [@petemounce](https://www.github.com/petemounce) 
+-  Peter Oliver [@mavit](https://www.github.com/mavit) 
+-  Peter Sprygada [@privateip](https://www.github.com/privateip) 
+-  Peter Tan [@tanpeter](https://www.github.com/tanpeter) 
+-  Philippe Makowski [@pmakowski](https://www.github.com/pmakowski) 
+-  Phillip Gentry, CX Inc [@pcgentry](https://www.github.com/pcgentry) 
+-  Quentin Stafford-Fraser [@quentinsf](https://www.github.com/quentinsf) 
+-  Ramon de la Fuente [@ramondelafuente](https://www.github.com/ramondelafuente) 
+-  Raul Melo [@melodous](https://www.github.com/melodous)  
+-  Ravi Bhure [@ravibhure](https://www.github.com/ravibhure) 
+-  René Moser [@resmo](https://www.github.com/resmo) 
+-  Richard Hoop [@rhoop](https://www.github.com/rhoop) 
+-  Richard Isaacson [@risaacson](https://www.github.com/)  
+-  Rick Mendes [@rickmendes](https://www.github.com/risaacson) 
```

Comment:
While you are doing this, can you fix my github link?

It should be https://github.com/rickmendes/. 

I didn't have proper permissions to push the change myself to your PR. Thanks.